src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -623,12 +623,16 @@ private void OnColumnsChanged()
         private DataFrame Sort(string columnName, bool isAscending)
         {
             DataFrameColumn column = Columns[columnName];
-            DataFrameColumn sortIndices = column.GetAscendingSortIndices();
+            PrimitiveDataFrameColumn<long> sortIndices = column.GetAscendingSortIndices(out Int64DataFrameColumn nullIndices);
+            for (long i = 0; i < nullIndices.Length; i++)
+            {
+                sortIndices.Append(nullIndices[i]);
```

Comment:
Is there an `AppendRange` so we don't have to loop over each one and append?

Reply:
Unfortunately, no. Does seem like a good API to add though.